<launch>
  <node pkg="nodelet" type="nodelet" name="nodelet_manager" args="manager" output="screen"/>

  <node pkg="nodelet" type="nodelet" name="nodelet1"
        args="load depth_image_proc/point_cloud_xyzrgb nodelet_manager" output="screen">
    <remap from="rgb/camera_info" to="/zedm/zed_node/depth/camera_info"/>
    <remap from="rgb/image_rect_color" to="/sem_image"/>
    <!-- <remap from="rgb/image_rect_color" to="/zedm/zed_node/left/image_rect_color"/> -->
    <remap from="depth_registered/image_rect" to="/zedm/zed_node/depth/depth_registered"/>
    <remap from="depth_registered/points" to="/semantic_pointcloud"/>
    <param name="queue_size" value="500"/>
  </node>
</launch>